Source/AtomicEditor/Editors/SceneEditor3D/SceneView3D.cpp

```diff
@@ -443,6 +447,13 @@ bool SceneView3D::OnEvent(const TBWidgetEvent &ev)
         if (ev.ref_id == TBIDC("close"))
             return false;
     }
+    if (ev.type == EVENT_TYPE_KEY_UP)
+    {
+        if (ev.special_key == TB_KEY_ESC)
+        {
+            sceneEditor_->GetSelection()->Clear();
+        }
```

Comment:
Does this clear selection if you are in the inspector in like a string field and hit esc?